dtformats/unified_logging.py

```diff
@@ -502,7 +539,8 @@ def _ReadChunkSet(self, file_object, file_offset, chunk_header):
             data_offset:data_offset + 16])
 
       chunkset_chunk_header = self._ReadStructureFromByteStream(
-          uncompressed_data, data_offset, data_type_map, 'chunk header')
+          uncompressed_data[data_offset:], data_offset, data_type_map,
```

Comment:
This is the workaround I added since _ReadStructureFromByteStream doesn't actually use data_offset to start further down a byte stream.

Reply:
correct data offset is used for debugging only